File: ABS/Java/ABC086A.java
import java.util.*;

class Main {
    public static void main(String args[]) {
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        int b = sc.nextInt();
        String s;
        if((a*b)%2 == 0)
            s = "Even";
        else
            s = "Odd";
        System.out.println(s);
    }
}

File: ABS/JavaScript/ABC086A.js
function main(input) {
    const args = input.split(' ');
    const a = args[0];
    const b = args[1];

    if((a*b)%2 !== 0)
        console.log('Odd');
    else
        console.log('Even');
}

main(require('fs').readFileSync('/dev/stdin', 'utf8'));